osu.Game/Screens/Menu/IntroWelcome.cs

```diff
@@ -0,0 +1,149 @@
+                    RelativeSizeAxes = Axes.Both
+                }, AddInternal);
+            }
+        }
+
+        public override void OnSuspending(IScreen next)
+        {
+            this.FadeOut(300);
+            base.OnSuspending(next);
+        }
+
+        private class WelcomeIntroSequence : Container
+        {
+            private Sprite welcomeText;
+
+            [BackgroundDependencyLoader]
+            private void load(TextureStore textures)
+            {
+                Origin = Anchor.Centre;
+                Anchor = Anchor.Centre;
+                Children = new Drawable[]
+                {
+                    new Container
+                    {
+                        Anchor = Anchor.Centre,
+                        Origin = Anchor.Centre,
+                        AutoSizeAxes = Axes.Both,
+                        Children = new Drawable[]
+                        {
+                            new Container
+                            {
+                                AutoSizeAxes = Axes.Both,
+                                Children = new Drawable[]
+                                {
+                                    new LogoVisualisation
+                                    {
+                                        RelativeSizeAxes = Axes.Both,
+                                        Anchor = Anchor.Centre,
+                                        Origin = Anchor.Centre,
+                                        Alpha = 0.5f,
+                                        AccentColour = Color4.DarkBlue,
+                                        Size = new Vector2(0.96f)
+                                    },
+                                    new Container
+                                    {
+                                        AutoSizeAxes = Axes.Both,
+                                        Children = new Drawable[]
+                                        {
+                                            new Circle
+                                            {
+                                                Anchor = Anchor.Centre,
+                                                Origin = Anchor.Centre,
+                                                Size = new Vector2(480),
+                                                Colour = Color4.Black
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    welcomeText = new Sprite
+                    {
+                        Anchor = Anchor.Centre,
+                        Origin = Anchor.Centre,
+                        Scale = new Vector2(0.3f),
+                        Width = 750,
+                        Height = 78,
+                        Alpha = 0,
+                        Texture = textures.Get(@"Welcome/welcome_text@2x")
```

Comment:
`@2x` shouldn't be here. everything in lazer is @2x.

Reply:
The texture doesn't display at all when I remove `@2x`, I'd have assumed first as well that it wouldn't be needed, but it's actually not visible on my side. 

`@2x` removed, so `textures.Get(@"Welcome/welcome_text")`
![dotnet_2020-06-13_16-28-06](https://user-images.githubusercontent.com/803255/84571342-e78f9a80-ad92-11ea-869e-2a4a0c367067.png)

`@2x` added
![dotnet_2020-06-13_16-27-39](https://user-images.githubusercontent.com/803255/84571340-e65e6d80-ad92-11ea-9eab-c209fd4fa4c0.png)